src/middlewared/middlewared/plugins/iscsi_/targets.py

```diff
@@ -19,6 +19,7 @@ class iSCSITargetModel(sa.Model):
     iscsi_target_name = sa.Column(sa.String(120), unique=True)
     iscsi_target_alias = sa.Column(sa.String(120), nullable=True, unique=True)
     iscsi_target_mode = sa.Column(sa.String(20), default='iscsi')
+    iscsi_target_auth_networks = sa.Column(sa.JSON(), default=[])
```

Comment:
`default` is not needed here. Also `default=[]` is dangerous for the same reason as in function args list.